controlplane/kubeadm/internal/workload_cluster_coredns_test.go

```diff
@@ -82,7 +82,7 @@ func TestUpdateCoreDNS(t *testing.T) {
 				Spec: corev1.PodSpec{
 					Containers: []corev1.Container{{
 						Name:  coreDNSKey,
-						Image: expectedImage,
+						Image: "k8s.gcr.io/some-folder/coredns:1.6.2",
```

Comment:
Why move this here instead of having it in the variable?

Reply:
The `expectImage` variable was used only here, and here it is not defining the expectedValue but the starting value of the test case.
Given that I introduced `expectImage` in the test case struct, I removed ^^ to avoid confusion